### PR TITLE
Add config for GEELANGF411 board

### PR DESCRIPTION
**Pull request by Linjieqiang**

### CODE REVIEW

File: configs/default/GEEL-GEELANGF411.config [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+set mag_bustype = I2C
+set mag_i2c_device = 1
+set baro_bustype = I2C
+set baro_i2c_device = 1
+set serialrx_provider = SBUS
+set blackbox_p_ratio = 16
+set blackbox_device = SPIFLASH
+set dshot_burst = AUTO
+set dshot_bitbang = OFF
+set motor_pwm_protocol = DSHOT600
```
Comment (haslinghuis) [inline configs/default/GEEL-GEELANGF411.config:100]: Please remove if not an AIO board

File: configs/default/GEEL-GEELANGF411.config [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+# pin B10: DMA1 Stream 1 Channel 3
+dma pin A00 0
+# pin A00: DMA1 Stream 2 Channel 6
+dma pin A02 0
+# pin A02: DMA1 Stream 0 Channel 6
+dma pin A08 0
+# pin A08: DMA2 Stream 6 Channel 0
+
+# feature
+feature -RX_PARALLEL_PWM
```
Comment (haslinghuis) [inline configs/default/GEEL-GEELANGF411.config:79]: Please remove. We have change the default to `RX_SERIAL`

File: configs/default/GEEL-GEELANGF411.config [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+resource MOTOR 4 B07
+resource MOTOR 5 A00
+resource MOTOR 6 B10
+resource PPM 1 A03
+resource LED_STRIP 1 A08
+resource SERIAL_TX 1 A09
+resource SERIAL_TX 2 A02
+resource SERIAL_TX 11 A00
+resource SERIAL_RX 1 A10
+resource SERIAL_RX 2 A03
```
Comment (haslinghuis) [inline configs/default/GEEL-GEELANGF411.config:18]: resource is already assigned to PPM 1

File: configs/default/GEEL-GEELANGF411.config [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+resource SERIAL_RX 11 B10
```
Comment (haslinghuis) [inline configs/default/GEEL-GEELANGF411.config:19]: resource is already assigned to MOTOR 6

File: configs/default/GEEL-GEELANGF411.config [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+# feature
+feature -RX_PARALLEL_PWM
+feature -AIRMODE
+feature RX_SERIAL
+feature SOFTSERIAL
+feature TELEMETRY
+feature OSD
+
+# serial
+serial 0 8192 115200 57600 0 115200
```
Comment (haslinghuis) [inline configs/default/GEEL-GEELANGF411.config:87]: Does this board has a TRAMP on board?

File: configs/default/GEEL-GEELANGF411.config [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+serial 1 64 115200 57600 0 115200
+
+# master
+set mag_bustype = I2C
+set mag_i2c_device = 1
+set baro_bustype = I2C
+set baro_i2c_device = 1
+set serialrx_provider = SBUS
+set blackbox_p_ratio = 16
```
Comment (haslinghuis) [inline configs/default/GEEL-GEELANGF411.config:96]: What's this?

File: .github/CODEOWNERS [hunk 7]
```diff
@@ -68,7 +68,7 @@
 /configs/default/FLMO-* @cvetaevvitaliy @betaflight/unified-target-administrators
 /configs/default/FOXE-* @Johnsnow-foxeer @betaflight/unified-target-administrators
 /configs/default/GEFP-* @x4FF3 @betaflight/unified-target-administrators
-/configs/default/GEPC-* @Linjieqiang @betaflight/unified-target-administrators
+/configs/default/GEEL-* @Linjieqiang @betaflight/unified-target-administrators
```
Comment (haslinghuis) [inline .github/CODEOWNERS:71]: This should be done in a separate PR.

**Review by haslinghuis — CHANGES_REQUESTED** [review]

**Comment by Linjieqiang** [comment]
@haslinghuis Thanks. I will fix them later.

File: configs/default/GEEL-GEELANGF411.config (hunk 1 above)
Comment (Linjieqiang) [reply configs/default/GEEL-GEELANGF411.config:100]: It's a AIO board. 
so keep them.

File: configs/default/GEEL-GEELANGF411.config (hunk 5 above)
Comment (Linjieqiang) [reply configs/default/GEEL-GEELANGF411.config:87]: Yes, it integrates.

File: configs/default/GEEL-GEELANGF411.config (hunk 6 above)
Comment (Linjieqiang) [reply configs/default/GEEL-GEELANGF411.config:96]: Removed "set blackbox_p_ratio = 16"

File: .github/CODEOWNERS (hunk 7 above)
Comment (Linjieqiang) [reply .github/CODEOWNERS:71]: OKAY

**Comment by Linjieqiang** [comment]
I have found your request to modify, please check, thanks. @haslinghuis

File: configs/default/GEEL-GEELANGF411.config [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+set ibata_scale = 100
+set beeper_inversion = ON
+set beeper_od = OFF
+set max7456_spi_bus = 2
+set dashboard_i2c_bus = 1
+set flash_spi_bus = 2
+set gyro_1_bustype = SPI
+set gyro_1_spibus = 1
+set gyro_1_sensor_align = CW180
+set gyro_1_align_yaw = 1800
```
Comment (KarateBrot) [inline configs/default/GEEL-GEELANGF411.config:107]:
```suggestion
set gyro_1_align_yaw = 1800

```
Missing end of line

**Review by KarateBrot — CHANGES_REQUESTED** [review]

**Review by DusKing1 — APPROVED** [review]

**Review by KarateBrot — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Comment by Asizon** [comment]
@Linjieqiang this target wont work without cpu definitions

**Pull request merged**